package.json

```diff
@@ -28,13 +28,13 @@
     "version": "yarn run build && git add -A"
   },
   "devDependencies": {
-    "@babel/cli": "7.13.14",
-    "@babel/core": "7.13.14",
+    "@babel/cli": "7.14.3",
+    "@babel/core": "7.14.3",
```

Comment:
Storybook 6.2.9 (`@storybook/addon-docs`) has build conflicts with babel-parser.

Reply:
@hzhu, rolling back `@babel/core` fixes the problem for now. Also, bumping `@storybook/*` to v6.3.0-alpha.45 allows the build to succeed. However, the storybook modification results in loads of warnings:
> The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding ["@babel/plugin-proposal-private-methods", { "loose": true }]